### DISCUSSION TITLE
Any example to deploy a project using docker?

### DISCUSSION THREAD

**Discussion by lkyhfx** [DavidVujic/python-polylith#141] [Q&A] [answered]
The Lego bricks idea behind Poly lith is awesome. Can anyone provide an example that deploys a project using Docker?

**Comment by DavidVujic** [comment, marked as answer]
Hi @lkyhfx,

I'm happy to hear that you like the idea of Polylith 😄 

There is an example here on how to run a service that is built using Polylith:
https://github.com/DavidVujic/python-polylith-example/tree/main/projects/my_fastapi_project
